src/components/ContractAddresses/ContractAddresses.tsx

```diff
@@ -1,105 +1,104 @@
-import React, { useState, useEffect, ReactNode } from "react";
+import React, { useState, useEffect } from "react";
 
-interface DataObject {
-  [key: string]: string;
-}
-
-interface NestedObject {
-  [key: string]: DataObject;
-}
-
-interface FetchedData {
-  [key: string]: NestedObject;
-}
-
-const edgeCases: { [key: string]: string } = {
-  ccm: "CCM",
-  zevm: "zEVM",
-  bsc: "BSC",
-};
-
-const addressesURL =
-  "https://raw.githubusercontent.com/zeta-chain/protocol-contracts/main/data/addresses.json";
-
-const DataFetch: React.FC = () => {
-  const [data, setData] = useState<FetchedData | null>(null);
+const ContractAddresses = () => {
+  const [activeTab, setActiveTab] = useState("testnet");
+  const [isLoading, setIsLoading] = useState(true);
+  const [groupedData, setGroupedData] = useState({ testnet: {}, mainnet: {} });
+  const testnetUrl =
+    "https://raw.githubusercontent.com/zeta-chain/protocol-contracts/feat/addresses/data/addresses.testnet.json";
```

Comment:
do you feel it's better a raw read rather than an npm?

Reply:
Just for consistency. We have a bunch of tables fetching data directly from ZetaChain's API. I don't have a strong preference, we can switch to npm for protocol contracts.

Reply:
Created an issue for this https://github.com/zeta-chain/docs/issues/256